lib/absinthe_relay_keyset_connection.ex

```diff
@@ -18,7 +18,10 @@ defmodule AbsintheRelayKeysetConnection do
       first: 10,
       after: "0QTwn5SRWyJNbyIsMjZd"
     },
-    %{unique_column: :id}
+    %{
+      unique_column: :id,
+      null_coalesce: %{name: ""}  # Handle NULL names
+    }
```

Comment:
Feel free to remove it for the simple example as the section below covers it

Reply:
Yes, please remove `null_coalesce` from this example

Reply:
Actually nevermind, I'll do that